packages/cli/src/commands/lockedgold/lock.ts

```diff
@@ -47,7 +47,9 @@ export default class Lock extends BaseCommand {
     for (const txo of txos) {
       await displaySendTx('relock', txo, { from: address })
     }
-    const tx = lockedGold.lock()
-    await displaySendTx('lock', tx, { value: lockValue.toFixed() })
+    if (lockValue.gt(new BigNumber(0))) {
```

Comment:
Shouldn't this already be covered by this check in line 33?

```.addCheck(`Value [${value.toFixed()}] is > 0`, () => value.gt(0))```

Reply:
Nevermind. I see now that `lockValue` could be 0 if the value to be locked is covered by `relockValue`.